hugegraph-common/src/main/java/org/apache/hugegraph/util/ExceptionUtil.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 HugeGraph Authors
```

Comment:
we need remove this line & check-then-remove in all other files(if copy from others)